test/jobs/cancel-stripe-subscription.js

```diff
@@ -1,20 +1,22 @@
-const { test, tearDown } = require('tap')
 const nock = require('nock')
-const worker = require('../../jobs/cancel-stripe-subscription')
 
 const dbs = require('../../lib/dbs')
+const removeIfExists = require('../helpers/remove-if-exists')
+
+const cancelStripeSubscription = require('../../jobs/cancel-stripe-subscription')
 
 nock.disableNetConnect()
 nock.enableNetConnect('localhost')
 
-test('Cancel Stripe Subscription', async t => {
+test('Cancel Stripe Subscription', async () => {
   const { payments } = await dbs()
-  t.plan(3)
+  expect.assertions(3)
 
   nock('https://api.stripe.com/v1')
     .delete('/subscriptions/345')
     .reply(200, () => {
-      t.pass('Stripe called')
+      // Stripe called
+      expect(true).toBeTruthy()
```

Comment:
maybe we should replace those  ` expect(true).toBeTruthy()` with `.pass`  from https://github.com/jest-community/jest-extended